sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample2_DetectLastPointAnomaly.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Azure;
```

Comment:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```
Above text is missing at the top of the file.

Reply:
Done.

---
In reply to: [481469259](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r481469259) [](ancestors = 481469259)